sdk/textanalytics/ai-text-analytics/test/textAnalyticsClient.spec.ts

```diff
@@ -519,6 +520,28 @@ describe("[AAD] TextAnalyticsClient", function() {
       // TA NER public preview currently supports only english
       assert.ok(results.slice(0, sliceSize).every(isSuccess));
     });
+
+    it("accepts domain filter", async () => {
+      const [result] = await client.recognizePiiEntities(
+        [
+          {
+            id: "0",
+            text: "I work at Microsoft and my phone number is 333-333-3333",
+            language: "en"
+          }
+        ],
+        { domainFilter: PIIEntityDomainType.PROTECTED_HEALTH_INFORMATION }
+      );
+      if (!result.error) {
+        assert.equal(result.entities.length, 1);
+        assert.equal(result.entities[0].text, "333-333-3333");
+        assert.equal(result.entities[0].category, "Phone Number");
+        assert.equal(
+          result.redactedText,
+          "I work at Microsoft and my phone number is ************"
```

Comment:
One thing that's confusing to me here is that the doc string above for the PHI domain says that it will only return information in the PHI domain, but this was always returned and I don't remember this being in their description of PHI the last time I learned about this feature.

Reply:
I am confused -- does this return only PHI or filter out PHI. Samples suggest it filters it out, but doc string suggests it filters out every thing else.

Reply:
this is good feedback on the param name. We named it filter so it only returns PHI entities (similar to how appconfig has their [filters](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_azure_appconfiguration_client.py#L153)). Do you think this could be solved by better documentation, or do you think having the `fitler` in the name is too confusing